jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -0,0 +1,619 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 2 - Core services
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Find the steps and prerequisites for the server v4.x installation.
+:icons: font
+:toc: macro
+:toc-title:
+
+// This doc uses ifdef and ifndef directives to display or hide content specific to Google Cloud Storage (env-gcp) and AWS (env-aws). Currently, this affects only the generated PDFs. To ensure compatability with the Jekyll version, the directives test for logical opposites. For example, if the attribute is NOT env-aws, display this content. For more information, see https://docs.asciidoctor.org/asciidoc/latest/directives/ifdef-ifndef/.
+
+Before you begin with the CircleCI server v4.x core services installation phase, ensure all <<phase-1-prerequisites#,prerequisites>> are met.
+
+.Installation Experience Flow Chart Phase 2
+image::server-install-flow-chart-phase2.png[Flow chart showing the installation flow for server 3.x with phase 2 highlighted]
+
+NOTE: In the following sections replace any items or credentials displayed between `< >` with your details.
+
+toc::[]
+
+CircleCI server is installed as a helm chart.
+
+[#create-a-namespace]
+== 1. Create a namespace
+Create a namespace to install the application into.
+
+[source,shell]
+----
+kubectl create ns circleci-server
```

Comment:
@circleci/server is this namespace up to the user, should it be shown like this: `<circleci-server>`?

Reply:
Customers can use any name for the `k8s namespace` But considering `circleci server` is going to be only application in K8s cluster, using `circleci-server` for k8s namespace is fine.